package.json

```diff
@@ -40,10 +40,7 @@
   "jest": {
     "transform": {
       "^.+\\.tsx?$": [
-        "ts-jest",
-        {
-          "isolatedModules": true
```

Comment:
option deprecated.. I've moved to tsconfig obviously that will now apply to production files but don't think that will cause any issues. It just seems to add [additional warnings](https://www.typescriptlang.org/tsconfig/isolatedModules.html).